Dockerfile

```diff
@@ -73,11 +73,26 @@ COPY docker-assets/initialize-httpd-auth.service /usr/lib/systemd/system/initial
 RUN  mkdir -p /etc/systemd/system/httpd.service.d
 COPY docker-assets/httpd-environment.conf /etc/systemd/system/httpd.service.d/environment.conf
 
+# Ruby 2.3
+ENV TERM=xterm \
+    LANG=en_US.UTF-8
+RUN yum -y install --setopt=tsflags=nodocs rh-ruby23
```

Comment:
Instead of using a separate ruby package (scl) can we base this image off of https://github.com/ManageIQ/container-ruby?

Reply:
I thought of that, but can we have 2 FROM lines ?  I really need FROM centos/httpd:latest more.

Reply:
Ah right, then can we at least maintain the same method of installing ruby as https://github.com/ManageIQ/container-ruby/blob/master/Dockerfile?

Reply:
Yep, that's fine.  My only requirement was ruby 2.2 for sinatra, so we should be ok there.
I'll give it a try.